tests/Transports.AspNetCore.Tests/WebSockets/WebSocketConnectionTests.cs

```diff
@@ -511,7 +511,29 @@ public async Task ExecuteAsync_WebSocket_EatsWebSocketExceptions()
                 throw new WebSocketException();
             })
             .Verifiable();
-        await _connection.ExecuteAsync(mockReceiveStream.Object);
+        await Should.ThrowAsync<OperationCanceledException>(() => _connection.ExecuteAsync(mockReceiveStream.Object));
```

Comment:
Test name contradicts with this line. No?

Reply:
I mean test did not throw before and it throws after.

Reply:
True. Now it rethrows as OCE

Reply:
Fixed (the name of the test)

Reply:
FYI OCEs were typically thrown before upon disconnection which were not caught. Once in a while a WebSocketException was thrown instead, which was caught. So it more of a bug fix to rethrow the WebSocketException as an OCE, not a breaking change.